MicroPython/scripts/generate_terraform.py

```diff
@@ -29,10 +33,28 @@ def save_terraform_output_as_file(_output_file_name: str):
         outfile.write(tf_output_str)
 
 
+def _check_terraform():
+    """Checks if terraform binary exists in the system"""
+
+    terraform_path = shutil.which(TERRAFORM_BINARY_NAME)
+    if terraform_path is not None:
```

Comment:
In the scenario, where user have both local terraform binary, and terraform installed in system, which one will be called?

Reply:
Terraform installed in the system